spec/features/user_adds_a_release_spec.rb

```diff
@@ -35,10 +84,21 @@
 
     expect(page).to have_content("Release added")
     expect(page).to have_content(band.name)
+    expect(page).to have_content("Edit Release")
+    expect(page).to have_content("Delete Release")
     expect(page).to have_content(50)
   end
 
-  scenario 'invalid information in form to add a release' do
+  scenario 'admin inputs invalid information' do
+    user = FactoryGirl.create(:admin)
+
+    visit new_user_session_path
+
+    fill_in 'Email', with: user.email
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.